idseq/locations.py

```diff
@@ -81,7 +81,7 @@ def set_location_matches(csv_data, matched_locations):
                 if value in matched_locations:
                     result = matched_locations[value]
                     is_human = any(
-                        [metadata.get(n) == "Human" for n in constants.HOST_GENOME_ALIASES]
+                        [(metadata.get(n)).lower() == "human" for n in constants.HOST_GENOME_ALIASES]
```

Comment:
Are locations trimmed to county in the cli?
Should we make sure that web recognizes human as the human host genome?

Reply:
Yes, `process_location_selection` removes the city name.
Good catch for the web app! I update the PR https://github.com/chanzuckerberg/idseq-web-private/pull/236 to make it case insensitive on the web as well.

Reply:
As a side note, eventually this all should be done on a single point on the web.